airbyte-integrations/connectors/source-sendgrid/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: fbb5fbe2-16ad-4cf4-af7d-ff9d9c316c87
-  dockerImageTag: 0.4.3
+  dockerImageTag: 0.4.4
```

Comment:
I think we can just skip pumping version here and just /approve-and-merge with comment after all other CI tests passed

Reply:
Gotcha thanks! Reverted all the version bumps